pizzaplace/services/yelp_api.py

```diff
@@ -0,0 +1,19 @@
+import requests
+from requests_oauthlib import OAuth1
+from meetup.services.generic_api import GenericApi
+from meetuppizza.settings import base
+
+class YelpApi(GenericApi):
```

Comment:
How is this abstraction working for you? I like the idea, but has the `GenericApi` base class been useful in any way so far?